### ISSUE TITLE
How-To use

### ISSUE THREAD

**Issue by chiefymuc** [gnumpi/esphome_matrixio#3]
I'd recommend to add the following to the readme:

Once connected to Home Assistant, you need to turn on the everloop light, say your command and turn off the everloop light again. Then your command will be processed by Home Assistant (of course assuming your Home Assistant Voice has been setup, e.g. following this guide: https://www.home-assistant.io/voice_control/thirteen-usd-voice-remote/

Took me far too long to figure this out :)

**Comment by gnumpi** [comment]
Thanks for your recommendation. You are right, some more information on how to setup the voice control with home assistant would be helpful. So far the configuration file was only meant to show how to include the matrixio into ESPHome.  Of course, binding the voice assistant to the everloop light was only a quick and dirty check if everything works. I checked in an updated configuration file which has a dedicated template switch for enabling the voice control independently of the leds.

**Comment by lostkohibu** [comment]
I have also a recommendation. For all those are not familiar with python an venv. Please add additionell hints to deal with the line esphome compile..... Cause the setup_esphome did not activate the virtual enviroment, so it was hard to figure out how it's work's.
Yes, you can see it in the script where the venv dir is locatet. An you can run the source command. But finally thanks for your repo.

**Comment by gnumpi** [comment]
thanks for your recommendation. Actually the setup script should activate the virtual environment, did you make sure to use 'source' ?
```bash
source scripts/setup_esphome.sh
```
Did this not work for you?

**Comment by lostkohibu** [comment]
yes i use the source command, but the venv was not activatet, no (venv) promp. It's okay, i learned so much ;-). Btw, should i build the firmware on a raspberry or better on a more powerfull hardware?

**Comment by gnumpi** [comment]
It's a bug then, thanks for reporting it. 

I always build the firmware on my macbook because it's much faster and more convenient if you use an IDE like Visual Studio Code. If you have enabled 'over the air' (ota) in your esphome configuration you can even update your esp32 flash via wifi. Only the initial flashing has to be done from the raspberry.
